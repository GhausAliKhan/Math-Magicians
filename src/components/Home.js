import React from 'react';

function Home() {
  return (
    <div>
      <section>
        <div className="container">
          <h1>Welcome to our Page!</h1>
          <p>
            <br />
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Debitis
            dignissimos iure soluta facere, odio quae, perspiciatis quisquam
            sint atque voluptatem blanditiis sunt iste laudantium rerum officia
            earum praesentium laboriosam. Odit accusamus consequatur quam?
            Deleniti quos tempore itaque cum consequuntur minus ratione omnis,
            quas qui nam facilis vero cumque officiis repellat?
            <br />
          </p>
          <p>
            <br />
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Explicabo
            quibusdam officia porro commodi repudiandae quae laborum libero
            ullam obcaecati voluptatem suscipit necessitatibus odit nemo debitis
            rerum nisi, nobis asperiores ut a quia animi doloribus!
            <br />
          </p>
        </div>
      </section>
    </div>
  );
}

export default Home;
