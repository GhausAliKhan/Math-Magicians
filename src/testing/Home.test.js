import { render } from '@testing-library/react';
import Home from '../components/Home';

describe('Renders Home Component', () => {
  test('render', () => {
    const { home } = render(<Home />);
    expect(home).toMatchSnapshot();
  });
});
