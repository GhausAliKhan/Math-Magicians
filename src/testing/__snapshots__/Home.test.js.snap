// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Home Component render 1`] = `undefined`;
